csv_importer/parsers.py

```diff
@@ -1,21 +1,33 @@
 import databpy as db
 import polars as pl
 import numpy as np
+import bpy
 
 
 def polars_df_to_bob(df: pl.DataFrame, name: str) -> db.BlenderObject:
     vertices = np.zeros((len(df), 3), dtype=np.float32)
     bob = db.create_bob(vertices, name=name)
 
+    update_bob_from_polars_df(bob, df)
+    return bob
+
+
+def update_obj_from_csv(obj: bpy.types.Object, csv_file: str) -> None:
+    bob = db.BlenderObject(obj)
+    df = pl.read_csv(csv_file)
+    if len(df) != len(bob):
+        bob.new_from_pydata(np.zeros((len(df), 3), dtype=np.float32))
```

Comment:
Will this preserve the applied geonodes modifiers?
If yes, then this is great! I'll test this in an hour or so.

Reply:
Yes it does. It changes the mesh data underneath, but keeps the Object and any modifiers on the object the same.